chpl/chpl-api/e2e/collections/surveillance-report-controller.postman_collection.json

```diff
@@ -1679,7 +1706,12 @@
 											"     pm.expect(pm.response.text()).to.include(\"SLI Compliance\");\r",
 											"     pm.expect(pm.response.text()).to.include(\"Q1\");\r",
 											"});\r",
-											""
+											"\r",
+											"pm.test(\"/surveillance-report/quarterly/{quarterlyReportId} response should have reactiveSurveillanceSummary and disclosureRequirementsSummary fields \", function () {\r",
+											"    var respbody = pm.response.json()\r",
+											"    pm.expect(respbody).to.have.property('reactiveSurveillanceSummary');\r",
```

Comment:
Why here it says respbody instead of respbody[0] like other added tests? (line #1647)

Reply:
Response of /quarterly/{quarterlyReportId} is not an array and hence using respbody instead of respbody[0]